components/sendinblue/actions/add-or-update-contact/add-or-update-contact.mjs

```diff
@@ -0,0 +1,101 @@
+import sendinBlueApp from "../../sendinblue.app.mjs";
+
+// legacy_hash_id: a_0Mi7n5
+export default {
+  key: "sendinblue-add-or-update-contact",
+  name: "Add or Updated a contact",
+  description: "Add or Updated a contact and list",
```

Comment:
"Add or Update"

Also, is the "and list" intentional? The key and name just say add or update contact, so the "list" stood out.